zilliqa/src/p2p_node.rs

```diff
@@ -112,6 +114,19 @@ impl P2pNode {
             )
             .map_err(|e| anyhow!(e))?,
             mdns: mdns::Behaviour::new(Default::default(), peer_id)?,
+            autonat: autonat::Behaviour::new(
+                peer_id,
+                autonat::Config {
+                    // Config changes to speed up autonat initialization.
+                    retry_interval: Duration::from_secs(10),
+                    refresh_interval: Duration::from_secs(30),
+                    boot_delay: Duration::from_secs(5),
+                    throttle_server_period: Duration::ZERO,
+                    // Don't attempt to reach IPs that are known to be at a private.
+                    only_global_ips: false,
```

Comment:
Doesn't your comment mean the opposite of the configuration?

If `only_global_ips` is `false`, I assume that means we allow private IPs, but your comment says the opposite?

Am I confused?

Reply:
No, I think I am. I'm going to remove it and set it to default, I suspect it's doing nothing anyway